osh/cmd_parse.py

```diff
@@ -2127,7 +2152,7 @@ def ParseCommand(self):
         # Top-level keywords to hide: func, data, enum, class/mod.  Not sure about
         # 'use'.
         if self.parse_opts.parse_tea():
-            if self.c_id == Id.KW_Func:
+            if self.c_id == Id.KW_Func:  # Duplicate kept for compatibility with tea
```

Comment:
Hm yeah we should decide whether to maintain this code, but it looks like it doesn't cause a big problem right now ... that was sort of an experiment to give the mycpp runtime a nicer syntax :)